Zend/zend_API.c

```diff
@@ -686,7 +694,7 @@ static const char *zend_parse_arg_impl(int arg_num, zval *arg, va_list *va, cons
 				}
 				if (!try_convert_to_string(arg)) {
 					*pce = NULL;
-					return "a valid class name";
+					return check_null ? "a valid class name or null" : "a valid class name";
```

Comment:
Side note: This error message doesn't make sense, because it's going to be printed as `of type a valid class name`. We can't actually hit this though, because `try_convert_to_string()` will throw on failure. Maybe use `return "";` here? Or `return "IRRELEVANT";` or something...

Reply:
Thanks! I think I like `return ""` + a short comment the most